### DISCUSSION TITLE
Tabellerarischer Fahrplan: Spalte Abhängigkeiten

### DISCUSSION THREAD

**Discussion by PianoRailways** [topological-state/stskit#40] [Q&A] [answered]
Für was stehen die Abhängigkeiten im Tabellenfahrplan?

- Signal(x) (wobei x Zahlen von 1 - 14? sind)
- Plan
- Ersatz
- Ankunft (Zugnamenummer, x)
- ... weitere?

**Comment by topological-state** [comment, marked as answer]
die abhängigkeiten zeigen an, nach welcher regel die abfahrtsverspätung an dem punkt berechnet wird. eine reihe von abhängigkeiten oder korrekturen (suche noch einen passenderen begriff) wird automatisch gesetzt, andere können vom fdl gesetzt werden (sobald implementiert).

- fix: manuelle korrektur durch fdl
- Signal: abfahrt ist wegen ausfahrtssignal x minuten verspätet
- Plan: zug soll planmässig abfahren und verspätung aufholen, wenn möglich
- Ersatz: zug bleibt am gleis stehen, bis er die nummer gewechselt hat
- Kupplung: verspätung übertragen beim kuppeln
- Flügelung: verspätung übertragen beim flügeln
- Ankunft: zug wartet auf ankunft eines anderen zuges und zusätzlich x minuten. wird bei folgezügen verwendet, um eventuelle verspätung zu übernehmen. kann vom fdl bei kreuzungen angewendet werden (noch nicht implementiert).
- Abfahrt: zug wartet auf abfahrt eines anderen zuges und zusätzlich x minuten. kann vom fdl bei überholungen angewendet werden (noch nicht implementiert).
